Contact/about.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta content="width=device-width, initial-scale=1.0">
+        <title>About our Replate Team</title>
+        <link href="about.css" rel="stylesheet"/>
+        <script src="Menu/menu.js" async></script>
+        <script src="Bio/bio.js" async></script>
+    </head>
+
+    <body>
+        <div class="header">
+            <img class="menu-button" src="../img/burger-icon.png"/>
+            <h1>About our Replate Team</h1>
+        </div>
+        <div class="menu">
+            <ul>
+                <li href="/index.html">Home</li>
```

Comment:
Having `/index.html` is likely to cause issues depending on how your site is deployed. Did you mean `../index.html`?